Automattic/jetpack
# Launchpad Checklist API: Update write flow task definitions

Fixes https://github.com/Automattic/jetpack/issues/30041 and https://github.com/Automattic/wp-calypso/issues/75504

## Proposed changes:

* Add task definitions to Launchpad Checklist API for write flow

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






1. Run `bin/jetpack-downloader test jetpack-mu-wpcom-plugin update/launchpad-write-flow-task-definitions` to build and sync this branch to your sandbox. 
2. Sandbox a test site. 
3. Start a new general onboarding site from https://wordpress.com/start, select the "Write & Publish" onboarding goal, and proceed to launchpad. 
4. Go to `https://developer.wordpress.com/docs/api/console/`, select Rest API and wpcom/v2, and input the following: `/sites/YOURSITESLUG/launchpad/checklist?checklist_slug=write`. Confirm you get back an appropriate checklist that matches the status of checklist items on Launchpad.
5. Verify that the `disabled` attribute is `false` for the `design_selected` task
6. Go back to the frontend launchpad screen from Step 3, and try completing one or more tasks. After each step, re-run the test in Step 4. You should alway see the correct task status in the developer console, matching the status of tasks on the frontend Launchpad.
